contracts/src/core/EigenDACertVerifier.sol

```diff
@@ -1,79 +0,0 @@
-// SPDX-License-Identifier: MIT
-pragma solidity ^0.8.9;
-
-import {EigenDACertVerifierV1} from "src/periphery/EigenDACertVerifierV1.sol";
-import {EigenDACertVerifierV2} from "src/periphery/EigenDACertVerifierV2.sol";
-import {IEigenDAThresholdRegistry} from "../interfaces/IEigenDAThresholdRegistry.sol";
-import {IEigenDABatchMetadataStorage} from "../interfaces/IEigenDABatchMetadataStorage.sol";
-import {IEigenDASignatureVerifier} from "../interfaces/IEigenDASignatureVerifier.sol";
```

Comment:
we may need this contract - or at least its binding in the interim given we'll potentially have to support cross compatibility with the older cert verifier interface within the V2 clients.... I think. 

thoughts @samlaf @litt3 ?

Reply:
Conduit just upgraded all their rollups to V2, and are using our currently deployed EigenDACertVerifier for v2. See https://eigenlabs.slack.com/archives/C05LFNLDN3W/p1745589330959239

Reply:
I'm not following you guys' discussion here btw. So let's discuss during standup. I agree with ethen that we need to be careful as we already have people like conduit running on testnet with the current CertVerifier, so proxy will have to keep bindings for it somewhere/somehow. Perhaps this should be part of larger discussion to move to a VerifyDACert signature that only takes bytes, and what's the best way to deal with this re version management, bindings generation, etc.

Reply:
@ethenotethan is this contract only used by arbitrum? I don't think any of eigenda v1 tests use it right..? Maybe mantle will need it but they can just copy-paste the "audited" version. So I think I agree we just need to keep the interface around to we can build bindings for it.